ts/data/data.ts

```diff
@@ -142,7 +142,9 @@ async function getAllConversations(): Promise<Array<ConversationModel>> {
   const conversationsAttrs =
     (await channels.getAllConversations()) as Array<ConversationAttributes>;
 
-  return conversationsAttrs.map(attr => new ConversationModel(attr));
+  return conversationsAttrs
```

Comment:
Great to have a defensive check here, just going to update it to be a little safer.